src/Components/Users/Users.module.css

```diff
@@ -0,0 +1,43 @@
+.badgeActiveInActive{
+  color: #ffffff;
+  font-family: "Montserrat-Regular", Helvetica;
+  font-size: 16px;
+  font-weight: 250;
+  border-radius: 15px;
+  text-align: center;
+
+}
+.active {
+  background-color: #009247;
+}
+
+.inactive {
+  background-color: #922e25b2;
+}
+
+
+.blockBtn {
+  color: #121010;
+  background: rgb(255, 255, 255);
+  cursor: pointer;
+  font-size: 16px;
+  text-align: center;
+  border: none;
+  border-radius: 15px;
+}
+
+.blockBtn:hover {
+  background: red;
```

Comment:
It's not advisable to use distinct background colors for the hover and normal states (e.g., white and red). A more visually appealing approach is to employ a consistent base color and its lighter variant. For instance, if the normal state background is red, consider using a lighter shade of red for the hover state.